src/flask_sqlalchemy/model.py

```diff
@@ -188,7 +294,11 @@ def should_set_tablename(cls: type) -> bool:
         return not (
             base is cls
             or base.__dict__.get("__abstract__", False)
-            or not isinstance(base, sa_orm.DeclarativeMeta)
+            or not (
```

Comment:
Maybe describe the different methods of inheritance:
* SQL 1 Meta
* SQL 2 DeclarativeBase
* SQL @ DeclarativeBaseNoMeta